controllers/userController.js

```diff
@@ -305,35 +304,21 @@ exports.readUser = async (req, res) => {
   }
 };
 
-//delete one user
-exports.deleteUser = async (req, res) => {
-  try{
-    const email = req.body.email;
-   
-    let verify = await this.verifyUser( email );
-
-    if (verify.NUM == 0) {
-      return res.send({ status: STATUS_CODE.INCORRECT_USER_EMAIL, result: email });
-    }
-
-    let result = await this.removeUser( email );
-
-    if (result.affectedRows){
-      return res.send({ status: STATUS_CODE.SUCCESS });
-    }
-  } catch (error) {
-    return res.send({ status: STATUS_CODE.ERROR, result: error });
-  }
-  return res.send({ status: STATUS_CODE.SUCCESS});
-}
+// return empty list when no value is found in DB
+// return the max Visions group number
+exports.visionsNums = async (req, res) => {
```

Comment:
Doesn't respond with the correct list of vision groups for the Visions Assignment page

Reply:
In addition, we need another API similar to visionsNums for the filter of VUceptor
<img width="358" alt="image" src="https://user-images.githubusercontent.com/107736505/197245153-7ba8b532-1537-4643-a9fc-a587c2cbfe74.png">